docker-compose.local.yaml

```diff
@@ -27,5 +27,39 @@ services:
     volumes:
       - .:/app
 
+  redis:
+    container_name: care_redis
+    image: redis:alpine
+    restart: always
+    ports:
+      - "6379:6379"
+
+  celery:
+    container_name: care_celery
+    build: 
+      context: .
+      dockerfile: docker/DevDockerfile
+    env_file:
+      - ./docker/.local.env
+    entrypoint: ["bash", "devCelery.sh"]
```

Comment:
Since this is the local version, we should mount local volumes as well.

Reply:
done